src/actor/model.rs

```diff
@@ -785,7 +810,8 @@ mod test {
                 .spawn_bfs()
                 .join()
                 .unique_state_count(),
-            2); // initial and delivery of Interesting
+            2
+        ); // initial and delivery of Interesting
```

Comment:
Not a blocker: comment is no longer associated w/ the line it references (and there are some other examples). Leaving this here as a reminder to correct later.

Reply:
Done in https://github.com/stateright/stateright/commit/1466b3ca650a53c6910107ae95500e9664c8ad50